velox/functions/sparksql/String.h

```diff
@@ -15,6 +15,11 @@
  */
 #pragma once
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "folly/ssl/OpenSSLHash.h"
+#pragma GCC diagnostic pop
```

Comment:
I don't really understand why this is still needed as folly's include dirs should be marked as `-isystem` and not produce errors... though I have seen it too...

Reply:
I was confused as well at first. This isn't when compiling folly though; this is when including these headers but compiling Velox's targets.